test/tools/ossfuzz/yulProto.proto

```diff
@@ -286,29 +286,29 @@ message AssignmentStatement {
 
 message IfStmt {
   required Expression cond = 1;
-  required Block if_body = 2;
+  required Block block = 2;
 }
 
 message BoundedForStmt {
-  required Block for_body = 1;
+  required Block block = 1;
 }
 
 message ForStmt {
-  required Block for_body = 1;
+  required Block block = 1;
   required Block for_init = 2;
   required Block for_post = 3;
   required Expression for_cond = 4;
 }
 
 message CaseStmt {
   required Literal case_lit = 1;
-  required Block case_block = 2;
+  required Block block = 2;
 }
 
 message SwitchStmt {
   required Expression switch_expr = 1;
   repeated CaseStmt case_stmt = 2;
-  optional Block default_block = 3;
```

Comment:
Perhaps `default_block` is more appropriate here? Is there a reason just `block` is better?

Reply:
Would it make it easier if we had something like
```
message DefaultBlock {
     required Block block;
}
```

Reply:
Not sure about the suggestion above suggestion. I feel that reading the spec after the `default_block -> block` would be slightly confusing. Maybe we can add a comment on top that this is the default statement?

Reply:
On second thought, this PR seems to make readability poor for avoiding typed if--else. Since the latter is already used in yul proto mutator, it shouldn't be much of an additional burden to add one such if-else block for extracting differently named block statements from statements.